source/extensions/omni.isaac.lab/omni/isaac/lab/sensors/imu/imu.py

```diff
@@ -0,0 +1,224 @@
+        """
+        # Initialize parent class
+        super()._initialize_impl()
+        # create simulation view
+        self._physics_sim_view = physx.create_simulation_view(self._backend)
+        self._physics_sim_view.set_subspace_roots("/")
+        # check if the prim at path is a rigid prim
+        prim = sim_utils.find_first_matching_prim(self.cfg.prim_path)
+        if prim is None:
+            raise RuntimeError(f"Failed to find a prim at path expression: {self.cfg.prim_path}")
+        # check if it is a RigidBody Prim
+        if prim.HasAPI(UsdPhysics.RigidBodyAPI):
+            self._view = self._physics_sim_view.create_rigid_body_view(self.cfg.prim_path.replace(".*", "*"))
+        else:
+            raise RuntimeError(f"Failed to find a RigidBodyAPI for the prim paths: {self.cfg.prim_path}")
+
+        # Create internal buffers
+        self._initialize_buffers_impl()
+
+    def _update_buffers_impl(self, env_ids: Sequence[int]):
+        """Fills the buffers of the sensor data."""
+        # check if self._dt is set (this is set in the update function)
+        if not hasattr(self, "_dt"):
+            raise RuntimeError(
+                "The update function must be called before the data buffers are accessed the first time."
+            )
+        # obtain the poses of the sensors
+        pos_w, quat_w = self._view.get_transforms()[env_ids].split([3, 4], dim=-1)
+        quat_w = math_utils.convert_quat(quat_w, to="wxyz")
+
+        # store the poses
+        self._data.pos_w[env_ids] = pos_w + math_utils.quat_rotate(quat_w, self._offset_pos_b)
+        self._data.quat_w[env_ids] = math_utils.quat_mul(quat_w, self._offset_quat_b)
+
+        # obtain the velocities of the link COM
+        lin_vel_w, ang_vel_w = self._view.get_velocities()[env_ids].split([3, 3], dim=-1)
+        # if an offset is present or the COM does not agree with the link origin, the linear velocity has to be 
+        # transformed taking the angular velocity into account
+        lin_vel_w += torch.linalg.cross(ang_vel_w, math_utils.quat_rotate(quat_w, self._offset_pos_b - self._com_pos_b[env_ids]), dim=-1)
+
+        # obtain the acceleration of the link COM
+        # lin_acc_w, ang_acc_w = self._view.get_accelerations()[env_ids].split([3, 3], dim=-1)
+        # # if an offset is present or the COM does not agree with the link origin, the linear acceleration has to
+        # # be transformed taking the angular velocity and acceleration into account
+        # lin_acc_w += torch.cross(ang_acc_w, math_utils.quat_rotate(quat_w, self._offset_pos_b - self._com_pos_b[env_ids]), dim=-1) + torch.cross(
+        #     ang_vel_w, torch.cross(ang_vel_w, math_utils.quat_rotate(quat_w, self._offset_pos_b - self._com_pos_b[env_ids]), dim=-1), dim=-1
+        # )
+        # numerical derivative
+        lin_acc_w = (lin_vel_w - self._prev_lin_vel_w )/ self._dt
+        ang_acc_w = (ang_vel_w - self._prev_ang_vel_w)/ self._dt
```

Comment:
```suggestion
        # numerical derivative
        lin_acc_w = (lin_vel_w - self._prev_lin_vel_w )/ self._dt
        ang_acc_w = (ang_vel_w - self._prev_ang_vel_w)/ self._dt
```
From what @Mayankm96 said in the dev meeting the get_accelerations function just does numerical differentiation any way. In an attempt to reduce a breaking version change we could just stick with the numerical derivative defined here instead of calling get_accelerations,.

Reply:
@pascal-roth any thoughts on what way to move forward for acceleration?